api/types/plugins/logdriver/entry.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
```

Comment:
As a general comment, everything in `api/types` is strictly the Request/Response types of the engine-api. The logger API doesnt belong in here. I think it should be moved out, maybe to a `pkg/logger` package?